### PR TITLE
Add passwords for various CDH versions and reports manager when needed

### PR DESCRIPTION
**Pull request by kirtiv1**

### CODE REVIEW

File: start.py [hunk 1]
```diff
@@ -77,8 +77,8 @@
 SSL_CLIENT_PASSWORD = 'clientpass'
 
 # Navigator related constants.
-NAVIGATOR_AUDIT_SERVER_POSTGRESQL_PASSWORD = 'kjIPHIDCuX'
-NAVIGATOR_METASERVER_POSTGRESQL_PASSWORD = '5lgfqqyGWP'
+# Following versions need to have Reports Manager role created when license is applied which is required for Navigator.
+CDH_VERSIONS_WITH_REPORTS_MANAGER_NEEDED = ['5.10.0', '5.11.1', '5.12.0', '5.13.0']
```
Comment (dimaspivak) [inline start.py]: Can we just check for 5.[10-13] without regard for the last digit?

File: start.py [hunk 2]
```diff
@@ -751,33 +751,72 @@ def _configure_spark2(deployment, cluster, history_server_node):
...
+    navigator_roles = []
+    if cdh_version in CDH_VERSIONS_WITH_REPORTS_MANAGER_NEEDED:
+        logger.info('Updating Reports Manager base role configs ...')
+        configs = {'headlamp_database_name': 'rman',
+                   'headlamp_database_user': 'rman'}
+        deployment.update_cm_service_role_config_group_config('mgmt-REPORTSMANAGER-BASE', configs)
+
+        logger.info('Configuring Reports Manager role ...')
+        # Fetch the embedded PostgreSQL database password created for Reports Manager role.
+        command = ("grep com.cloudera.cmf.REPORTSMANAGER.db.password /etc/cloudera-scm-server/db.mgmt.properties | "
```
Comment (dimaspivak) [inline start.py]: Let's replace grep | awk with `Node.get_file` and then grok it with Python code.

File: start.py [hunk 3]
```diff
@@ -751,33 +751,72 @@ def _configure_spark2(deployment, cluster, history_server_node):
...
-                                        'value': NAVIGATOR_AUDIT_SERVER_POSTGRESQL_PASSWORD},
+                                        'value': navigator_dbpassword},
                                        {'name': 'navigator_database_type',
                                         'value': 'postgresql'}
                                    ]},
                                    'hostRef': {'hostId': primary_node.host_id}}
-    deployment.create_cm_roles([navigator_metadata_server_role, navigator_audit_server_role])
+    navigator_roles.append(navigator_audit_server_role)
+
+    logger.info('Adding Navigator related roles to cluster (%s) ...', DEFAULT_CLUSTER_NAME)
```
Comment (dimaspivak) [inline start.py]: Super nit: just say "Adding Navigator roles to cluster" instead of "Navigator related"

**Review by dimaspivak — CHANGES_REQUESTED** [review]
Just nits. Yay Navigator support! :)

File: start.py (hunk 2 above)
Comment (kirtiv1) [reply start.py]: Done.

File: start.py (hunk 3 above)
Comment (kirtiv1) [reply start.py]: Done.

File: start.py (hunk 1 above)
Comment (kirtiv1) [reply start.py]: Done.

**Review by dimaspivak — APPROVED** [review]
Lovely!

**Pull request merged**